### PR TITLE
Update cognite-sdk  to 4.0.1

### PR DESCRIPTION
**Pull request by emelvarol-cognite**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "cognite-extractor-utils"
-version = "3.0.0-beta"
+version = "4.0.0-beta"
```
Comment (einarmo) [inline pyproject.toml:3]: I'd prefer `3.0.0-beta2`, or something along those lines. We're already on a beta of a major version, no need to skip 3.0.0 entirely.

Comment (emelvarol-cognite) [reply pyproject.toml:3]: Sure, I also wasn't sure about the versioning here. I guess it doesn't affect the users in a way that requires major version change. 🤔

Comment (emelvarol-cognite) [reply pyproject.toml]: Is it better now?

**Review by einarmo — APPROVED** [review]
LGTM, thanks

**Pull request merged**